src/Calendar.js

```diff
@@ -31,8 +32,9 @@ export default class Calendar extends Component {
 
   handleDateSelected(evt) {
     const dateBefore = !isBefore(evt.target.dataset.date, new Date(format(this.props.minDate, 'MM/DD/YYYY')))
+    const dateAfter = !isAfter(evt.target.dataset.date, new Date(format(this.props.maxDate, 'MM/DD/YYYY')))
 
-    if (dateBefore) {
+    if (dateBefore && dateAfter) {
```

Comment:
I think this is wrong.  I would think the following scenarios maybe:
```
dateBefore -> set date to evt.target.dataset.date
dateAfter -> set date to dateAfter - 1
```

Either way your conditional should at the very least be:
```
if (dateBefore || dateAfter) {
```

Reply:
Yeh, it can't be both before and after